packages/react-components/react-datepicker-compat/src/components/DatePicker/useDatePickerStyles.styles.ts

```diff
@@ -57,6 +62,7 @@ export const useDatePickerStyles_unstable = (state: DatePickerState): DatePicker
       datePickerClassNames.popupSurface,
       popupSurfaceClassName,
       state.popupSurface.className,
+      inlinePopup && styles.inline,
```

Comment:
I considered also applied this style when `mountNode` is set but left that off as it seems like that style should be applied to the mount node itself.